pkg/util/net/ip/ip.go

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+package ip
+
+import (
+	"io/ioutil"
+	"net"
+
+	netutils "k8s.io/utils/net"
+)
+
+const (
+	// sysfs
+	DISABLE_IPV6_PATH = "/proc/sys/net/ipv6/conf/default/disable_ipv6"
+
+	IPV4_LOOPBACK_ADDRESS = "127.0.0.1"
```

Comment:
I think that even for constants the capital letter applies and the convention of camel-case over underscores. No?

Reply:
this is the convention i saw on other files of the project
i have go lint installed, 
which say
```
don't use ALL_CAPS in Go names; use CamelCasego-lint
```
but generally we dont obey go lint atm

so question if we use convention or create a file which is little different

Reply:
"Because others do it like that" is not a good reasoning.
Technically, you should obey the convention and make sure you do not export constants when it is not needed.